staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
Good! Let's take suggestion from sttts
Your concern also makes sense to me @crassirostris 
Frankly speaking, I am just a newbie without much work experience, so I'd prefer to take suggestions from you guys and learn from you guys.